### ISSUE TITLE
Robotolight Font Not Working Properly Out of the Box

### ISSUE THREAD

**Issue by ptsouchlos** [posquit0/Awesome-CV#440]
I'm having issues with the changes in 1a5691e63f0c59bb49dfbcebbf733ae691d42c6f. It seems that the `robotolight` font is not being found or doesn't exist. 

https://github.com/posquit0/Awesome-CV/blob/5b05d935658377c73a3456269d23a276b19804f7/awesome-cv.cls#L165

**Comment by telescopicSum** [comment]
Hi, 

I am just a basic user. I am trying to use the docker command to compile my cv. I encounter the similar problem, it gives me below output on Ubuntu prompt. The roboto.sty is the issue.
-----------------------------------
..
..
(/usr/share/texlive/texmf-dist/tex/latex/fontawesome5/fontawesome5.sty
(/usr/share/texlive/texmf-dist/tex/latex/fontawesome5/fontawesome5-utex-helper.
sty (/usr/share/texlive/texmf-dist/tex/latex/fontawesome5/tufontawesomefree.fd)
(/usr/share/texlive/texmf-dist/tex/latex/fontawesome5/tufontawesomebrands.fd)))

### ! LaTeX Error: File `roboto.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

Enter file name: 
! Emergency stop.
<read *> 
         
l.78 \RequirePackage
                    [default,opentype]{sourcesanspro}^^M
No pages of output.
Transcript written on examples/cv.log.
make: *** [Makefile:16: cv.pdf] Error 1
Error: Command 'make' returned '2' when executed with arguments ''.
Now exiting container.
...
-----------------------------
Please advise.

**Comment by rauerhans** [comment]
I think it's because the font used to be bundled with this repo, but the latex distribution inside of the container image does not have the package

**Comment by rauerhans** [comment]
yup, using [this image](https://hub.docker.com/r/thomasweise/docker-texlive-full) fixes it (albeit it's quite a bloated image, literally a full latex with gb of layers, but it works). E.g. replace the docker command with this should fix it for you:
```docker run --rm --user $(id -u):$(id -g) -i -w "/doc" -v "$PWD":/doc thomasweise/docker-texlive-full make```

**Comment by telescopicSum** [comment]
Thank you for your follow-up. Yes, I read i can use the texlive-full but did not figure out the command. I will take note of above.

It was about 2 months ago, I read that it is related to the change of #403 . So at the end after many hours to searching and trying, i was inspired from the Overleaf website, copy the "fonte" folder (my "fix" was just to re-create the pre-403 version, I think) 

![Screenshot from 2022-11-06 11-20-53](https://user-images.githubusercontent.com/114092072/200182509-2973378f-3b20-4e53-b628-722d91e8f76b.png)

and it was one big step forward for me towards successful compilation, plus some changes on latex script on fontawsome*.tff

It is working now for my need. It is my first experience with Docker.

**Comment by Moanrisy** [comment]
Reverting to commit #403 and add fonts dir solved this issue

**Comment by vimkin** [comment]
> Reverting commit [#403](https://github.com/posquit0/Awesome-CV/pull/403) and add fonts dir solved this issue

I'd vote for keeping it as a package dependency, but we should update the README to clarify that it's not included in the `thomasweise/texlive` image. Users have then 2 options to address this then:
- Manually install the Roboto font in their Docker container
- Switch to `thomasweise/docker-texlive-full`, which includes all fonts but requires 2 GB of disk space.

There is also a hint to map local fonts to the Docker container as described [here](https://github.com/thomasWeise/docker-texlive?tab=readme-ov-file#1-usage). However, this didn't work for me. Maybe someone finds a way to make it work with `-v /path/to/fonts/:/usr/share/fonts/external/`.

**Comment by OJFord** [comment]
I'd suggest using OJFord/docker-texlive, it's updated weekly in a GitHub Actions cron job, and it's what I use myself, so slightly more tested in that sense. The thomasweise images seem not to have been updated for several years.
